patches/additionalPythonPackages/default.nix

```diff
@@ -518,33 +518,48 @@ in
   };
 
   equation = pythonPackages.buildPythonPackage rec {
+    pname = "Equation";
     version = "1.2.01";
-    name = "Equation-${version}";
+    name = "${pname}-${version}";
 
-    src = pkgs.fetchurl {
-      url = "https://pypi.python.org/packages/fd/0d/ede829e7c0c457b651de2792cd19a739e4885477f59832da54d2cc7a1982/Equation-${version}.tar.gz";
+    src = pythonPackages.fetchPypi {
```

Comment:
I have not seen fetchPypi used in this file.
Is there any specific reason to avoid it?
Seems to simplify porting PyPi-based packages otherwise in comparsion with pkgs.fetchFromGitHub or pkgs.fetchurl.

Reply:
This file has been written when `fetchPypi` did not exist and installation from wheel was a pain. But since the last Nix release, using`fetchPypi` it is the recommended way.

Reply:
OK, thanks for clarifying that.